test/connectFactoryObservable.test.tsx

```diff
@@ -1,14 +1,21 @@
 import { connectFactoryObservable } from "../src"
-import { from, of, defer, concat, BehaviorSubject } from "rxjs"
-import { renderHook, act } from "@testing-library/react-hooks"
+import { from, of, defer, concat, BehaviorSubject, throwError } from "rxjs"
+import { renderHook, act as actHook } from "@testing-library/react-hooks"
+import { render, act } from "@testing-library/react"
+import { map, switchMap } from "rxjs/operators"
+import { Component, ErrorInfo, FC } from "react"
+import React from "react"
 
 const wait = (ms: number) => new Promise(res => setTimeout(res, ms))
 
 describe("connectFactoryObservable", () => {
   const originalError = console.error
   beforeAll(() => {
     console.error = (...args: any) => {
-      if (/Consider adding an error/.test(args[0])) {
+      if (
+        /Uncaught 'controlled error'/.test(args[0]) ||
```

Comment:
That's interesting - React stills throws the error (so it shows up as an uncaught exception in the console) even if there's an ErrorBoundary. Not only that, but CRA will overlay a ugly error message on the whole body if it's in dev mode, telling you that you should fix that error that happened.

It only uses the ErrorBoundary to handle that specific component crashing, and in prod it doesn't show the overlayed message.